.travis.yml

```diff
@@ -1,12 +1,20 @@
 language: node_js
 node_js:
-- '6'
-- '6.1'
-- '5.11'
+  - "6"
+  - "6.1"
+  - "5.11"
+
+services:
+  - mongodb
+  before_script:
```

Comment:
There is a yam syntax error here. `before_script` and services should start from the same line
```yml
services:
  - mongodb
before_script:
```